examples/nextjs-babel/src/i18n.ts

```diff
@@ -14,7 +15,25 @@ export async function loadCatalog(locale: string) {
 export function useLinguiInit(messages: Messages) {
   const router = useRouter()
   const locale = router.locale || router.defaultLocale!
-  i18n.loadAndActivate(locale, messages, false)
+  const isClient = typeof window !== "undefined"
+
+  if (!isClient && locale !== i18n.locale) {
+    // there is single instance of i18n on the server
+    // note: on the server, we could have an instance of i18n per supported locale
+    // to avoid calling loadAndActivate for (worst case) each request, but right now that's what we do
+    i18n.loadAndActivate({ locale, messages })
```

Comment:
Arguments should not be an object here correct?

Reply:
they should: https://github.com/lingui/js-lingui/blob/998996381f5e5a458c2eccf650f949d8c5d8ac89/packages/core/src/i18n.ts#L187

Reply:
Yes, I just rechecked, apologies. Nice job 💯